src/nvim/message.c

```diff
@@ -3114,14 +3114,14 @@ int vim_vsnprintf(char *str, size_t str_m, char *fmt, va_list ap, typval_T *tvs)
 {
   size_t str_l = 0;
   bool str_avail = str_l < str_m;
-  char        *p = fmt;
+  char *p = fmt;
   int arg_idx = 1;
 
-  if (p == NULL)
+  if (!p)
```

Comment:
I'm on the opposite camp. Specially for conditions on pointer, I much prefer the abbreviated form.
If I recall correctly, abbreviated form is preferred style in neovim.
@justinmk Am I right on this?